R/pt.R

```diff
@@ -147,21 +158,26 @@ pt <- function(iso = TRUE
                 , pts.50 = 4
                 , pts.dialysis = 0.1
                 , pts.age = 4
-                , n = 2){
+                , n = 2
+                , check.validity = TRUE){
+  
+  if(check.validity){
+    candidate_dataframe_check(candidates)
+  }
 
   n <- max(1, n)
 
   data <- cp(data = data) %>% # Isto pode ser feito antes do for loop de candidato vs dador
```

Comment:
This can be deleted from here and the cp() function should be returned to Lima file.